packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -92,6 +92,7 @@ export interface TreeBlock extends ProductBlockInstance {
     label: string | number | boolean;
     callback: () => void;
     children: TreeBlock[];
+    outsideCurrentSubscription: boolean;
```

Comment:
Suggesting naming this `isOutsideCurrentSubscription`

Reply:
Done